### DISCUSSION TITLE
Can't Install multiple version of OGR_FDW for PostgreSQL v14.x and v15.x on RHEL 8

### DISCUSSION THREAD

**Discussion by ayrus02** [pramsey/pgsql-ogr-fdw#247] [Q&A]
Hi, 

I'm currently want to upgrade my existing database from PostgreSQL v14.x to v15.x, for the installation of PostgreSQL, PostGIS, PgRouting, everything was fine (I install both version 14 and version 15 in the same machine). But when I install ogr_fdw_15 packaged, it is conflicted with ogr_fdw_14, specifically on this file:

Error: Transaction test error:
  file /usr/lib/.build-id/a0/a43182caa55ed9d3562e2d9b74a1dda150e2c4 from install of ogr_fdw_15-1.1.3-1.rhel8.x86_64 conflicts with file from package ogr_fdw_14-1.1.3-1.rhel8.x86_64

How can I install multiple ogr_fdw version package on the same machine?

Regards,
Surya

**Comment by pramsey** [comment]
No idea, seems like a weird packaging thing. It *should* be possible with good packaging to have an ogr_fdw_14 and ogr_fdw_15 simultanteously. Please take this up with the packager.
